components/modals/settings/schemas.tsx

```
@@ -0,0 +1,367 @@
+    MinusIcon,
+    OctagonAlert,
+    PencilIcon,
+    PlusIcon,
+    RotateCw,
+    SaveIcon,
+    TrashIcon
+} from "lucide-react"
+import { ChangeEvent, useCallback, useContext, useEffect, useMemo, useState } from "react"
+import { Popover, PopoverContent, PopoverTrigger } from "@/components/ui/popover"
+import { SchemaWorker } from "@/components/providers/schema-worker-provider"
+import { Error } from "@/components/error"
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)